odkx-src/build-scripts.rst

```diff
@@ -67,6 +67,15 @@ If you are on :program:`Windows` use :file:`gradlew.bat` instead.
 
   If you are building with Android Studio, you will need to select the correct build variant. This is important when you don't have androidlibrary or androidcommon in your :ref:`build-scripts-directory-structure`. These are discussed more in the :ref:`next section <build-scripts-flavors>`.
 
+.. warning::
+
+  The Android tools have a release signature key unique to the ODK-X applications. Communication between the applications is secured by this key and can only happen when the application keys match. This ensures that the information passing between the applications cannot be intercepted by another application.
+  You can find more information on Android app signing `here <https://developer.android.com/studio/publish/app-signing>`_.
+  
+  When you compile the Android tools locally, they get assigned a new key called a default debug key. Your compiled application will only be able to communicate with another locally compiled application because their default debug keys are the same.
+  
+  It is important to know this because your locally compiled application cannot communicate with other ODK-X Android apps unless the apps are locally compiled.
```

Comment:
In addition, the key is also read by the Google Maps library. The signing key is tied to the Google Maps API key. The Google Maps API key included in the repo is tied to the ODK-X signing key.